vscode.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo "VScode install initiated!!!"
+
+install_app(){
+
+	echo "Installing requried additional utilities"
+	sudo apt update &&
+		sudo apt install software-properties-common apt-transport-https wget
+
+	echo "Importing GPG key and enbling the repository"
+	wget -q https://packages.microsoft.com/keys/microsoft.asc -O- | sudo apt-key add -
+	sudo add-apt-repository "deb [arch=amd64] https://packages.microsoft.com/repos/vscode stable main"
+
+	echo "install VScode"
+	sudo apt update &&
+		sudo apt install code
+}
+
+###Main
+
+install_app
```

Comment:
![vscode_terminal](https://user-images.githubusercontent.com/58501088/122680837-f82ce900-d20e-11eb-9d45-0d9b2b0c4a31.png)

Reply:
- Got the above o/p. 
- Add a condition to check whether the tools and packages exists.